backend/src/meetings/meetings/meetings.controller.ts

```diff
@@ -25,4 +30,31 @@ export class MeetingsController {
       tags: tags
     }).then(meetings => meetings.map(MeetingEntity.toDTO));
   }
-}
+
+  @Get('attending/confirmed/:username')
+  async findMeetingsAttendedByUser(@Param('username') username : string): Promise<MeetingDto[]>{
+    const userID = username;
+    if (userID != username){
+      throw new ForbiddenException('You do not have permission to access this data');
+    }
+    return this.meetingsService.getMeetingsForUserWhichTookPart(username).then(username => username.map(MeetingEntity.toDTO));
+  }
+
+  @Get('attending-information/:username')
+  async findAttendingInformationForUser(@Param('username') username : string): Promise<BookingDto[]>{
+    const userID = username;
+    if (userID != username){
+      throw new ForbiddenException('You do not have permission to access this data');
+    }
+    return this.meetingsService.getAttendingInformationForUser(username).then(username => username.map(BookingEntity.toDTO));
+  }
+
+  @Get('author/:username')
+  async findMeetingsByAuthor(@Param('username') username : string): Promise<MeetingDto[]>{
+    const userID = username;
+    if (userID != username){
+      throw new ForbiddenException('You do not have permission to access this data');
+    }
+    return this.meetingsService.getMeetingsFromAuthor(username, userID).then(username => username.map(MeetingEntity.toDTO));
```

Comment:
userID and username will always be the same. The equals check later will always be true.  The idea was that you pass the username from the EvendemyRequest object because it is the verified username of the requesting user.